data_crawler/T20/match_ids.csv

```diff
@@ -0,0 +1,7525 @@
+5531,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+2979,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3612,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3750,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3755,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3894,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3893,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3510,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+5120,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3658,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+4021,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+4794,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+5256,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3944,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+3412,Matches/MatchScorecard_T20.asp?MatchCode=0962,2019-10-18
+5546,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5540,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5536,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5537,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5538,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5539,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5540,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5541,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5542,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5543,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5544,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5546,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5545,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5523,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5518,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5547,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5519,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5522,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5521,Matches/MatchScorecard_T20.asp?MatchCode=0965,2019-10-18
+5529,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5011,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5007,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5009,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5003,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5526,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5527,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5007,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5009,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5011,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5013,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5529,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18
+5530,Matches/MatchScorecard_T20.asp?MatchCode=0964,2019-10-18

```

Comment:
I might be wrong, does the `name` column contain `player_ids`?

Reply:
oh yea

Reply:
should if fix that?

Reply:
No Not neccessary as of now, Until we filter old matches, Let be there